samples/hello_everybody/hello_everybody.c

```diff
@@ -104,6 +105,7 @@ static int init_hello(void)
 
 	inode_lock(data_file->d_inode);
 	int attr_assign_ret = notify_change(&nop_mnt_idmap, data_file, &data_attr, NULL);
+	/* what it is used for? why debugfs_create_file(...0644) is not enough? */
```

Comment:
?

Reply:
I wanted to explicitly set file uid. It's possible that when I create file, its uid is already `GLOBAL_ROOT_UID`, and if so - I did a lot of useless work. But anyway now I know how to do it)